ti/store/params.js

```diff
@@ -31,7 +29,9 @@ export default {
     },
     mutations: {
         setParams (state, payload) {
-            state.params = payload
+            for (let a in payload) {
+                state[a] = payload[a]
```

Comment:
You might want to add a payload.hasOwnProperty(a) here.
Also is a Vue.set(state, a, payload[a]) might be needed.

Reply:
done